external/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisInputDStream.scala

```diff
@@ -193,6 +197,21 @@ object KinesisInputDStream {
     }
 
     /**
+     * Sets the Kinesis initial position data to the provided timestamp.
+     * Sets InitialPositionInStream to [[InitialPositionInStream.AT_TIMESTAMP]]
+     * and the timestamp to the provided value.
+     *
+     * @param timestamp Timestamp to resume the Kinesis stream from a provided
+     *                  timestamp.
+     * @return Reference to this [[KinesisInputDStream.Builder]]
+     */
+    def withTimestampAtInitialPositionInStream(timestamp: Date) : Builder = {
```

Comment:
@brkyvz 
`withInitialPositionAtTimestamp` is an enhancer method for the InitialPositionAtTimestamp. If provided It will set the timestamp value along with the InitialPosition.AT_TIMESTAMP.

Its optional, hence the `initialPositionInStream` can still be used. This will not introduce and incompatibilities in usage. 
Thoughts ?